src/app/exercise/editors/editors.component.ts

```diff
@@ -20,6 +20,29 @@ export class EditorsComponent {
     this.debug = true;
   }
 
+  ngOnInit() {
+    this.files.map( file => {
+      file.opened = !file.readonly;
+    });
+  }
+
+  closeTab(file) {
+    file.opened = false;
```

Comment:
Why is there `file.opened` and `isOpenFile(file)` but they have different logic?